pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-matrix/default.nix

```diff
@@ -4,6 +4,8 @@ stdenv.mkDerivation rec {
   pname = "purple-matrix-unstable";
   version = "2019-06-06";
 
+  buildFlags = [ "CC=cc" ];
```

Comment:
Please add a comment explaining why this is necessary. People looking at this in the future may not know that this fixes the darwin build, and therefore just remove it (since the linux build works without that line). Or worse, they may never remove it even when it is not necessary anymore because they do not know what it does.

Also it looks like this is something that should get fixed upstream. It would be nice to create an upstream issue/PR and link that in the comment.